ddtrace/internal/telemetry/data/request_body.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+import json
+from typing import Optional
+
+from ...runtime import get_runtime_id
+from .application import Application
+from .host import Host
+from .payload import Payload
+
+
+class RequestBody:
```

Comment:
other than `payload` all of this data is basically static? can we optimize this in any way?